yjit/src/stats.rs

```diff
@@ -391,6 +390,9 @@ fn rb_yjit_gen_stats_dict() -> VALUE {
 
         // Live pages
         hash_aset_usize!(hash, "live_page_count", cb.num_mapped_pages() - freed_page_count);
+
+        // Code GC count
+        hash_aset_usize!(hash, "code_gc_count", CodegenGlobals::get_code_gc_count());
```

Comment:
> we probably shouldn't be adding state variables in many different places for different stats. We should probably refactor the code in stats.rs instead to differentiate between stats that we only collect in a stats build because they are expensive to collect vs stats that we always collect.

Agreed :+1: